tooling/reproducible/comparable_patch.sh

```diff
@@ -478,6 +478,43 @@ function neutraliseReleaseFile() {
   fi
 }
 
+# The last four bytes of a .gnu_debuglink ELF section contains a
+# 32-bit cyclic redundancy check (CRC32) of the separate .debuginfo
+# file.  A given .debuginfo file will differ when compiled in
+# different environments.  Specifically, if some declaration in a
+# system header is referenced by OpenJDK source code, and that
+# declaration's line number changes between environments, the
+# .debuginfo files will have many bytewise differences.  Even
+# seemingly inconsequential header changes will result in large
+# .debuginfo differences, for example, additions of new preprocessor
+# macros, or comment additions and deletions.  The CRC32 is thus
+# sensitive to almost any textual changes to system headers.  This
+# function changes the four bytes to zeroes.
+function neutraliseDebuglinkCRCs() {
+  if [[ "$OS" =~ CYGWIN* ]] || [[ "$OS" =~ Darwin* ]]; then
+    # Assume Cygwin and Darwin toolchains do not produce .gnu_debuglink sections.
+    return
+  fi
+  elf_magic="^7f454c46$"
+  # Does not handle filenames with newlines because the hexdump format does not support \0. 
+  find wipe -type f \! -name '*.debuginfo' -print -exec hexdump -n 4 -e '4/1 "%.2x" "\n"' '{}' ';' \
```

Comment:
```suggestion
  find "${JDK_DIR}" -type f \! -name '*.debuginfo' -print -exec hexdump -n 4 -e '4/1 "%.2x" "\n"' '{}' ';' \
```

Reply:
Oops, that was my testing directory. Force-pushed a fix.